app/src/components/ProjectView/ProjectView.js

```diff
@@ -27,11 +34,18 @@ graphComponent.inputMode = new GraphEditorInputMode();
 //Style:
 graph.nodeDefaults.size = new Size(150, 150);
 
-var loaded = false;
+// Construct a some sample nodes:
+const rootNode = generateNewNode(700,100, 'PROJECT-X')
```

Comment:
Die example nodes sollten nur vom Backend kommen. Ich würd die alle hier rauslöschen und dann gerne in der Datenbank die dummy daten ändern

Reply:
okay :)